.github/workflows/main.yml

```
@@ -11,6 +11,10 @@ name: main
 
 jobs:
   build:
+    if: >-
+      (github.event_name != 'schedule') ||
+      (github.event_name == 'schedule' &&
+      github.repository == 'planetarium/libplanet')
```

Couldn't this be simplified like below?:

```suggestion
    if: github.event_name != 'schedule' || github.repository == 'planetarium/libplanet'
```